src/component/Layout/MainLayout/MainLayout.jsx

```diff
@@ -8,16 +9,22 @@ import Footer from '../../ui/footer/Footer';
  * @param {object} props - react props with children
  * @returns {object} the component
  */
-const MainLayout = ({ children }) => (
-  <div>
-    <Header />
-    {children}
+export const MainLayout = ({ children, auth = {} }) => (
+  <Fragment>
+    <Header isAuthenticated={auth.isAuthenticated} username={auth.user.user.username} />
+    <main role="main" className="pt-5">
+      <div className="container-fluid">{children}</div>
+    </main>
```

Comment:
@henperi, please why is the `MainLayout` component connected to redux store? Also passing `auth.user.user.username` to the header, will raise an error if the user is not authenticated and why are we having`main`with a padding. Also, what is the purpose of the `container-fluid` div in all of our pages?